zerver/migrations/0268_add_userpresence_realm_timestamp_index.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-02-08 20:34
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0267_backfill_userpresence_realm_id'),
+    ]
+
+    operations = [
+        migrations.AlterIndexTogether(
+            name='userpresence',
+            index_together=set([('realm', 'timestamp')]),
```

Comment:
This should be a plain list so it stays ordered, right? It'd have the wrong meaning if comes out in a different order.

(I guess with Python 3.6+ a `set` will also get iterated in order. But it seems clearer to leave it as a list.)

Reply:
Hmm.  This code is automatically generated by Django, so it would be reasoanble to assume that it does the right thing.  Certainly the index it generates is this:
```
    "zerver_userpresence_realm_id_timestamp_25f410da_idx" btree (realm_id, "timestamp")
```
But I don't see why it's doing the right thing.

Reply:
Oh, as Steve points out, it's a set of tuples.

Reply:
Oh, I see -- a set of just one tuple. With all the nested enclosures, I lost count. :upside_down_face: Cool, then.

Perhaps this would be a bit easier to read as `{('realm', 'timestamp')}`, for that reason. But if Django is auto-generating this, there isn't much to be gained from trying to fiddle with that.